jobs/redis/templates/config/redis.conf.erb

```diff
@@ -105,9 +105,19 @@ databases 16
 #
 #   save ""
 
-save 900 1
-save 300 10
-save 60 10000
+<% if_p('redis_save_intervals') do
+    redis_save_intervals = p('redis_save_intervals')
+
+    if redis_save_intervals.length > 0
```

Comment:
@drnic the ```save ""``` command is not fundamental, meaning the resulting behaviour would be unchanged if we leave it out completely.

I just thought it would make it more evident, that disabling the snapshotting was intentional (to somebody who reads the resulting conf file) than simply omitting the line.